### PR TITLE
www-apache/mod_wsgi: add python 3.10

### PR DESCRIPTION
**Pull request by t0b3**
Closes: https://bugs.gentoo.org/829359

### CODE REVIEW

File: www-apache/mod_wsgi/files/mod_wsgi-4.7.1-py310.patch [hunk 1]
```diff
@@ -0,0 +1,125 @@
+From b439f1c411a9479ccc03c16465cdff50fede79d3 Mon Sep 17 00:00:00 2001
+From: Petr Viktorin <encukou@gmail.com>
+Date: Thu, 10 Jun 2021 15:45:03 +0200
+Subject: [PATCH] Use Py_CompileString rather than
```
Comment (djc) [inline www-apache/mod_wsgi/files/mod_wsgi-4.7.1-py310.patch:4]: Where did this come from? Maybe put a link in there?

**Review by djc — COMMENTED** [review]
Would like to verify the provenance of the patch here, otherwise seems fine.

File: www-apache/mod_wsgi/files/mod_wsgi-4.7.1-py310.patch (hunk 1 above)
Comment (t0b3) [reply www-apache/mod_wsgi/files/mod_wsgi-4.7.1-py310.patch:4]: It's simply backported from upstream https://github.com/GrahamDumpleton/mod_wsgi/commit/b439f1c411a9479ccc03c16465cdff50fede79d3
How would you like to mention it? Your suggestion is very welcome...

Comment (djc) [reply www-apache/mod_wsgi/files/mod_wsgi-4.7.1-py310.patch:4]: Putting that link in the body of the patch email (that is, between the `Subject` header and the `---` preceding the diffstat), should work fine.

**Comment by t0b3** [comment]
@djc thanks for your pragmatic proposition. Can you add the link on my
behalf? I'm only on emails these weeks. This would be perfect.

Dirkjan Ochtman ***@***.***> wrote on Sun., 19. Dez. 2021,
20:52:

**Review by djc — APPROVED** [review]

**Comment by djc** [comment]
LGTM, thanks!

**Comment by djc** [comment]
Rebased as e0ec91c1c0a09edb5c9d4b2e9b81d256e2be5d71, thanks!

**Pull request closed without merging**